fflas-ffpack/utils/test-utils.h

```diff
@@ -39,15 +39,14 @@
 #include <givaro/givinteger.h>
 #include <givaro/givintprime.h>
 #include <givaro/givranditer.h>
+#include <givaro/givtimer.h>
 
 #include <random>
 #include <functional>
 
 namespace FFLAS {
     uint64_t getSeed(){
-        struct timeval tp;
-        gettimeofday(&tp, 0) ;
-        return static_cast<uint64_t> (tp.tv_usec + tp.tv_sec*1000000);
+        return Givaro::BaseTimer::seed();
```

Comment:
Ok to rely on `Givaro::BaseTimer::seed` but this one currently only uses `tp.tv_usec` which is a positive integer < 10^6 i.e. only 20 bits.
We should add + `tp.tv_sec*1000000` to it.

Reply:
or use C11  tv_nsec, and  tv_nsec